packages/components/src/Well/Well.module.scss

```diff
@@ -1,69 +1,104 @@
-@import "~@kaizen/design-tokens/sass/color";
-@import "~@kaizen/design-tokens/sass/border";
-@import "~@kaizen/design-tokens/sass/spacing";
-
 .container {
-  border-width: $border-width-1;
-  border-radius: $border-solid-border-radius;
-  margin-bottom: $spacing-md;
+  border-width: var(--border-width-1);
+  border-radius: var(--border-solid-border-radius);
+  margin-bottom: var(--spacing-md);
+  color: var(--color-purple-800);
 }
 
 .noMargin {
   margin: 0;
 }
 
-// Background Style Variations
-.default {
-  background-color: $color-gray-300;
-  border-color: $color-gray-500;
+.white {
+  background-color: var(--color-white);
+  border-color: var(--color-gray-500);
+}
+
+.gray {
+  background-color: var(--color-gray-200);
+  border-color: var(--color-gray-500);
+}
+
+.blue {
+  background-color: var(--color-blue-100);
+  border-color: var(--color-blue-400);
+}
+
+.yellow {
+  background-color: var(--color-yellow-100);
+  border-color: var(--color-yellow-700);
+}
+
+.orange {
+  background-color: var(--color-orange-100);
+  border-color: var(--color-orange-300);
```

Comment:
```suggestion
  border-color: var(--color-orange-600);
```

Reply:
weird, though I updated this, maybe lost in a rebase